File: 268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int orig = 0;
        int asum = 0;
        for(int i=1;i<=n;i++){
            orig+=i;
            asum += nums[i-1];
        }
        return orig-asum;

    }
};

File: 268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n=len(nums)
        orignal_sum=(n*(n+1))/2
        array_sum=sum(nums)
        print(f"n:{n},orig_sum:{orignal_sum},arr_sum:{array_sum}")
        return int(orignal_sum-array_sum)